pkgs/top-level/aliases.nix

```diff
@@ -989,7 +989,7 @@ mapAliases {
   gdmd = throw "gdmd has been removed from Nixpkgs, as it depends on GDC which was removed"; # Added 2025-08-08
   gdome2 = throw "'gdome2' has been removed from nixpkgs, as it is umaintained and obsolete"; # Added 2024-12-29
   geocode-glib = throw "'geocode-glib' has been removed, as it was unused and used outdated libraries"; # Added 2025-04-16
-  geos_3_11 = throw "geos_3_11 has been removed from nixpkgs. Please use a more recent 'geos' instead."; # Added 2024-11-29
```

Comment:
The old alias should be kept as well.

Reply:
I'm not sure if there's an alias removal schedule, but as far as I know most old aliases are kept around.

Reply:
Ok removed.

But to be honest, it looks more like nobody took the time to do it than a real reason not to, to be honest. Imo they should be kept at least during 1 release cycle, but should be removed afterwards. There are indeed very few throw removals in the git historic, but when they happen it's mostly in this timeframe...